book/content/rdm/fairprinciples.md

```diff
@@ -63,7 +63,7 @@ should also regularly be updated as the research project progresses or diverge f
 Although started by a community operating in the life science, the FAIR principles have rapidly been adopted by
 publishers, funders, and pan-disciplinary infrastructure programmes and societies, in all disciplines. Many groups and
 organization are working to define guidance and tools to help researchers, as well as other stakeholders (librarians,
-funders, publishers, trainers etc.) to make data FAIR and assess its FAIRness level.
+funders, publishers, trainers et cetera) to make data FAIR and assess its FAIRness level.
```

Comment:
Is *et cetera* necessarily any better than *etc*? Maybe "and so on", or better"for example/such as" at the start of the sentence, which flags up the non-exhaustive nature of the list up front?

Reply:
Actually, let me read the style guide 😂

Reply:
Updated in 356576b